src/CommandBusInterface.php

```diff
@@ -2,7 +2,7 @@
 
 namespace ZfrCommandBus;
 
-interface CommandBusInterface
+interface CommandBusInterface extends MessageBusInterface
```

Comment:
the "dispatch" should return typehint void?

Reply:
This project is still at 7.0 yet, so we don't have void typehint.